<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1112</width>
    <height>858</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>16</width>
      <height>171</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="edPathCTxt">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>521</width>
      <height>22</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="maxLength">
     <number>1</number>
    </property>
    <property name="placeholderText">
     <string>Transition Token (leave empty for Îµ)</string>
    </property>
   </widget>
   <widget class="QComboBox" name="fromEdAddCom">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>50</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>From Node</string>
    </property>
   </widget>
   <widget class="QComboBox" name="toEdAddCom">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>50</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>To Node</string>
    </property>
   </widget>
   <widget class="QPushButton" name="adEdgeBtn">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>80</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Transition</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>1111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="delNodeCom">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Pick a Node to Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delNodeBtn">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>130</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Node</string>
    </property>
   </widget>
   <widget class="QComboBox" name="delEdCom">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>130</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Pick an Edge to Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delEdgeBtn">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>130</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Edge</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>1111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="searchBtn">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>180</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
   <widget class="QPushButton" name="adAccNodeBtn">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Accepted State</string>
    </property>
   </widget>
   <widget class="QPushButton" name="adTransNodeBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Transitional State</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Make q0 (starting state) an accepted state</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
